pkg/workflows/workload/upgrade_test.go

```diff
@@ -135,6 +135,11 @@ func (c *upgradeTestSetup) expectWriteWorkloadClusterConfig(err error) {
 	)
 }
 
+func (c *upgradeTestSetup) expectAWSIAMAuthKubeconfig(err error) {
```

Comment:
I am wondering if we call this "without" or "no" to make it clear that it is indeed not trying to call AWSIAMSuccess
```suggestion
func (c *upgradeTestSetup) expectWithoutAWSIAMAuthKubeconfig(err error) {
```

Reply:
Makes sense, updated this.